src/initialization/InitDistribution.cpp

```diff
@@ -97,16 +105,24 @@ namespace impactx
         std::string particle_type;  // Particle type
         pp_dist.get("particle", particle_type);
 
-        amrex::ParticleReal qm = 0.0; // charge/mass ratio (q_e/eV)
+        amrex::ParticleReal qe;     // charge (elementary charge)
+        amrex::ParticleReal massE;  // MeV/c^2
         if(particle_type == "electron") {
-            qm = -1.0/0.510998950e6;
+            qe = -1.0;
+            massE = 0.510998950;
         } else if(particle_type == "proton") {
-            qm = 1.0/938.27208816e6;
+            qe = 1.0;
+            massE = 938.27208816;
         }
-        else {
-            qm = 0.0;
+        else {  // default to electron
```

Comment:
Wouldn't I be able to set up muons, for instance, if I provided a name and `qe` and `massE` for them?

Reply:
yes, we did not add that case yet but could issue a warning, for instance.